mep/common/admin.py

```diff
@@ -92,17 +97,31 @@ def after_import(self, dataset, result, using_transactions, dry_run, **kwargs):
 
         # report how many need indexing
         logger.debug(
-            f"indexing {len(self.objects_to_index)} objects, dry_run = {dry_run}"
+            f"requesting index of {len(self.objects_to_index)} objects, dry_run = {dry_run}"
         )
 
+        # warn if too many
+        max2index = self.max_objects_to_index
+        num2index = len(self.objects_to_index)
+        if max2index and num2index > max2index:
+            messages.warning(
+                self.request,
+                f"The number of updated records in need of indexing ({num2index:,})"
+                f" exceeds the maximum amount indexable from the web interface ({max2index:,})."
+                f" Please be aware that you will need to manually re-index this table on the server.",
```

Comment:
I don't think we should use the word "table" here. We also don't have any way for a data editor / web admin to kick off a reindex... We may have a nightly cron job still running to index, not sure (and not sure if we want to rely on that).

Can we make the message more concise? Updated N records, indexed the first 100; ... [whatever action we want to recommend]

Reply:
Hm, what action do you want to recommend, then?